ngraph/test/frontend/shared/include/cut_specific_model.hpp

```diff
@@ -16,6 +16,9 @@ struct CutModelParam {
     std::vector<std::string> m_newInputs;
     std::vector<std::string> m_oldOutputs;
     std::vector<std::string> m_newOutputs;
+    std::string              m_tensorValueName;
```

Comment:
I believe we should consider to create separate test files for this, like 'tensor_value.cpp'. Per my understanding, this test is not related to 'cutting model' functionality.

Reply:
It is related to cutting the model. When you specify a tensor value in the middle of the model you cut everything that was before this tensor.